tests/schemas/test_property_schema.py

```diff
@@ -62,16 +62,50 @@ def test_manager_update(self, create_property, create_property_manager):
         pm_2 = create_property_manager()
         pm_3 = create_property_manager()
         payload = {"propertyManagerIDs": [pm_2.id, pm_3.id]}
+        context = {"request_type": "PUT", "name": prop.name}
 
-        PropertyModel.update(schema=PropertySchema, id=prop.id, payload=payload)
+        PropertyModel.update(
+            schema=PropertySchema, id=prop.id, payload=payload, context=context
+        )
         db.session.rollback()
 
         assert prop.managers == [pm_2, pm_3]
 
     def test_property_update_without_managers(self, create_property):
         prop = create_property()
         payload = {
-            "name": "The New Portlander Delux Apartment Complex Multnomah Suite Express"
+            "name": "The New Portlander Delux Apartment Complex Multnomah Suites",
         }
+        context = {"request_type": "PUT", "name": prop.name}
 
-        assert PropertyModel.update(schema=PropertySchema, id=prop.id, payload=payload)
+        assert PropertyModel.update(
+            schema=PropertySchema, id=prop.id, payload=payload, context=context
+        )
+
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestUpdatePropertyValidations:
+    def test_property_can_update_with_same_name(self, create_property):
```

Comment:
These look good. I don't think they need to be in a separate class though. Let's move them to the validation section of the class `TestPropertyManagerValidations` -> and while your in there rename that class name to `TestPropertyValidations`